mitodl/bootcamp-ecommerce
# Update validation email and create profile title

#### What are the relevant tickets?
Closes #662 
Closes #665

#### What's this PR do?
Tweaks the verification email text and the title for the account creation profile pages.

#### How should this be manually tested?
Create a new account and read the verification email.  It should conform to the example text in issue #662

Click on the email link, the next two pages should display a title of "Create Profile"

#### Screenshots 
<img width="661" alt="Screen Shot 2020-06-16 at 10 18 58 AM" src="https://user-images.githubusercontent.com/187676/84787166-a75f3080-afbb-11ea-8db2-26a613bb5eb3.png">

<img width="799" alt="Screen Shot 2020-06-16 at 10 57 50 AM" src="https://user-images.githubusercontent.com/187676/84795456-84397e80-afc5-11ea-8b19-75672db81732.png">

<img width="780" alt="Screen Shot 2020-06-16 at 10 58 06 AM" src="https://user-images.githubusercontent.com/187676/84795469-88659c00-afc5-11ea-8362-07c2aa795e29.png">
